api/v1beta1/hyperconverged_types.go

```diff
@@ -272,6 +272,11 @@ type HyperConvergedSpec struct {
 	// +kubebuilder:default=false
 	// +default=false
 	EnableApplicationAwareQuota *bool `json:"enableApplicationAwareQuota,omitempty"`
+
+	// LiveUpdateConfiguration holds cluster configuration for live update features - max cpu sockets,
```

Comment:
nit: "features" - should be "feature"?

Can you please rephrase to have something more expressive? e.g. "holds the cluster configuration for live update of virtual machines" - not sure my suggestion is accurate, but this is the kind of how I would like it to look like - not as a pipe to KubeVirt, but a configuration of the KubeVirt deployment with HCO.